src/test/java/com/browserstack/examples/config/Capabilities.java

```diff
@@ -0,0 +1,31 @@
+package com.browserstack.examples.config;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import lombok.Data;
+import lombok.ToString;
+
+/**
+ * Created with IntelliJ IDEA.
```

Comment:
I think we should remove these auto-generated comments.

Reply:
Auto-Generated Comments. Removed